includes/payment-methods/class-upe-payment-method.php

```diff
@@ -270,9 +270,10 @@ public function get_payment_token_for_user( $user, $payment_method_id ) {
 	/**
 	 * Returns testing credentials to be printed at checkout in test mode.
 	 *
+	 * @param string $account_country The country of the account.
 	 * @return string
 	 */
-	abstract public function get_testing_instructions();
+	abstract public function get_testing_instructions( string $account_country = null );
```

Comment:
I find it a little bit confusing to type the argument's `$account_country` as `string` (both with PHP 7.0 typing and with the PHP docblock), but setting `null` as its default value.

A few questions:
- In which scenario would `$account_country` be `null` or need a default value? `get_account_country()` will return `US` in case of `null` value: https://github.com/Automattic/woocommerce-payments/blob/206225842252ff39d290e5f9999cb544b41fcd2c/includes/class-wc-payments-account.php#L2300-L2300
- Should `null` be also typed in the docblock?

Reply:
I agree, no nullable default is needed. I removed it in 5712fbb8bbdcb937080d33148e0e779a76addbc3